<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>P_RmmvOperateBoard</class>
 <widget class="QWidget" name="P_RmmvOperateBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>文件</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>工程路径</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_path">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QToolButton" name="toolButton_select">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>选择工程</string>
        </property>
        <property name="icon">
         <iconset resource="../../../drillgifactionsequenceeditor.qrc">
          <normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/nav/Open_rmmv.png</normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/nav/Open_rmmv.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>工程名</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_name">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>操作</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QToolButton" name="toolButton_import">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>从rmmv中导入数据</string>
        </property>
        <property name="icon">
         <iconset resource="../../../drillgifactionsequenceeditor.qrc">
          <normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/nav/Import_rmmv.png</normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/nav/Import_rmmv.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_save">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>保存数据到rmmv</string>
        </property>
        <property name="icon">
         <iconset resource="../../../drillgifactionsequenceeditor.qrc">
          <normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/nav/Save_rmmv.png</normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/nav/Save_rmmv.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_software">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>打开rmmv编辑器</string>
        </property>
        <property name="icon">
         <iconset resource="../../../drillgifactionsequenceeditor.qrc">
          <normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/nav/Open_rmmv.png</normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/nav/Open_rmmv.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_run">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>运行rmmv</string>
        </property>
        <property name="icon">
         <iconset resource="../../../drillgifactionsequenceeditor.qrc">
          <normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/nav/Run_rmmv.png</normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/nav/Run_rmmv.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>修改plugins.js文件前，自动备份bak文件</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../drillgifactionsequenceeditor.qrc"/>
 </resources>
 <connections/>
</ui>
